src/th/reapertrans/src/eu/kanade/tachiyomi/extension/th/reapertrans/ReaperTrans.kt

```diff
@@ -0,0 +1,12 @@
+package eu.kanade.tachiyomi.extension.th.reapertrans
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class ReaperTrans : MangaThemesia(
+    "ReaperTrans",
+    "https://www.reapertrans.com",
```

Comment:
The `www` subdomain redirects to the naked domain. Change to avoid additional redirects (also in `build.gradle`).